.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Don't put this please an article on gitbook should be good to explain how to add this to your .gitignore-global